resources/views/alert.blade.php

```diff
@@ -6,7 +6,7 @@
     <div class="alert-content-wrapper alert-{{ $type }}-content @if($large ?? false) alert-content-large @endif">
         @isset($title)<span class="alert-{{ $type }}-title">{{ $title }}</span>@endif
         @isset($message)
-            <span class="block">{{ $message }}</span>
+            <span class="block break-all">{{ $message }}</span>
```

Comment:
`break-all` will mess up the flow of actual sentences. Use `break-words` instead